cogs/source.py

```diff
@@ -2,17 +2,27 @@
 from discord.ext import commands
 import os
 import spacy
+from pathlib import Path
+from typing import Dict, Final, List, Any
+import yaml
 
 class SourceCog(commands.Cog):
-    """A Discord cog for replying to 'source that' message and displaying the content of the most relevant file."""
+    """
+    A Discord cog for replying to 'source that' message and displaying the content of the most relevant file.
+    """
+    CONFIG_PATH : Final[str] = Path("./BOT_CONFIG.yaml")
+    KEY_PHRASE : Final[str] = 'source that'
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
```

Comment:
Make all of these variables Final and UPPERCASE

Also document these variables

nlp needs to have a better name, its not clear what it is.

Reply:
Made them Final and UPPERCASE with extra documentation.